babel.config.js

```diff
@@ -1,10 +1,13 @@
 module.exports = function (api) {
   api.cache(true)
 
-  const plugins = []
-  if (process.env.ISTANBUL === 'true') {
+  let plugins = ['babel-plugin-jsx-remove-data-test-id']
```

Comment:
Maybe it's less complex if we return different config for CYPRESS and without CYPRESS? Easier to read